<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetrePrincipale</class>
 <widget class="QMainWindow" name="FenetrePrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Videoclub</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="btnDeconnexion">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #444444;  /* gris foncé */
    color: pink;                /* texte blanc */
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #555555;  /* légèrement plus clair au survol */
}

</string>
      </property>
      <property name="text">
       <string>Déconnexion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableClients">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prénom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Courriel</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableFilms">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Titre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Durée</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Catégorie</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAjouterClient">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #444444;  /* gris foncé */
    color: white;                /* texte blanc */
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #555555;  /* légèrement plus clair au survol */
}
</string>
        </property>
        <property name="text">
         <string>Nouveau Client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnModifier">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #444444;  /* gris foncé */
    color: white;                /* texte blanc */
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #555555;  /* légèrement plus clair au survol */
}
</string>
        </property>
        <property name="text">
         <string>Modifier Client</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnAjouterFilm">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #444444;  /* gris foncé */
    color: white;                /* texte blanc */
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #555555;  /* légèrement plus clair au survol */
}
</string>
      </property>
      <property name="text">
       <string>Nouveau Film</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="btnSupprimer">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #444444;  /* gris foncé */
    color: red;                /* texte blanc */
    font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #555555;  /* légèrement plus clair au survol */
}
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
